RELEASENOTES-1.4.docu

```diff
@@ -257,4 +257,8 @@
       now by default log one <tt>spec-violation</tt> message on log level 1
       on first such write, and no logging on further such writes.
   </add-note></build-id>
+  <build-id value="next"><add-note> Added simple C99-style designated
+      initializers. Use together with <tt>struct</tt>, <tt>extern struct</tt>,
+      <tt>layout</tt>, and <tt>bitfields</tt> types are supported
+      <bug number="SIMICS-17252"/>.</add-note></build-id>
```

Comment:
What is the BKM for where to put the </ad-note></build-id> to avoid merge conflicts? last line of releasenote message or new line?

Reply:
The BKM is to *not* have it on a new line. Otherwise, after two simultaneous notes, a merge tool will propose that the identical closing tags should be merged into one.

So, it's a minor cost, and it's fine to have closing line on a separate line now and then (e.g. if that's what you get from auto-line-wrapping).